build/windows/gh.wxs

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<?ifndef ProductVersion?>
+<?error ProductVersion property not defined?>
+<?endif?>
+
+<!-- Define a unique UpgradeCode per platform -->
+<?define UpgradeCode=8CFB9531-B959-4E1B-AA2E-4AF0FFCC4AF4?>
+
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+    <Product Id="*" Name="GitHub CLI" Version="$(var.ProductVersion)" Language="1033" Manufacturer="GitHub, Inc." UpgradeCode="$(var.UpgradeCode)">
+        <Package Compressed="yes" InstallerVersion="200" InstallScope="perMachine"/>
```

Comment:
Cross-scoping like that isn't actually easy nor was it ever truly supported in practice. That feature predates UAC and even then barely worked. Security boundaries introduced with UAC broke what little support that feature had.

The problem is that in a per-user install the package would have to use not only a different target directory but also a registry path. Both can conditionalized but, with the same component GUIDs, if both MSIs are installed side-by-side it breaks a number of features. Best to either have separate MSIs, or completely separate features conditioned on install context. With WiX at least the gh.exe binary won't be duplicated, though it may look like it (separate entries in the embedded CAB actually point to the same blob - a trick we did in WiX for Visual Studio over a decade ago).

Reply:
Resolving per [comment](https://github.com/cli/cli/pull/4276#issuecomment-919005994)